app/api/dao/user.py

```diff
@@ -129,7 +130,9 @@ def update_user_profile(user_id, data):
             user.resume_url = data['resume_url']
 
         if 'photo_url' in data and data['photo_url']:
-            user.photo_url = data['photo_url']
+            image_type = data['photo_url'].split(';')[0].split('/')[1]
+            photo_url = aws_upload.upload_to_aws(data['photo_url'], os.environ.get("AWS_PROFILE_BUCKET"), username+"."+image_type)
```

Comment:
username + "." + image_type
Need to add the spaces between operators, and maybe a better way to handle this string append.

Reply:
Here, i'm thinking of assigning a new variable with username + "." + image_type and using variable as a parameter in function.